### ISSUE TITLE
  Undefined offset: -1 for some strings 

### ISSUE THREAD

**Issue by ahmedkheikal** [khaled-alshamaa/ar-php#6]
I'm getting  Undefined offset: -1 for some strings at ar-php/src/Arabic.php:2268 with no special cases for strings with errors

code sample

```php
        $Arabic = new Arabic();

        $p = $Arabic->arIdentify($html);

        for ($i = count($p) - 1; $i >= 1; $i -= 2) {
            $utf8ar = $Arabic->utf8Glyphs(substr($html, $p[$i - 1], $p[$i] - $p[$i - 1]));
            $html = substr_replace($html, $utf8ar, $p[$i - 1], $p[$i] - $p[$i - 1]);
        }
```

**Comment by khaled-alshamaa** [comment]
Dear @ahmedkheikal, it will be more helpful if you can share with us a sample of the $html variable that reproduces this error message, so we can trace back the source of such issue and fix it.

**Comment by khaled-alshamaa** [comment]
I believe [this commit](https://github.com/khaled-alshamaa/ar-php/commit/968968b65116e010b61061101fd84afecab2491e#diff-6dbe7dcbc53b188c0e4e2a74cd1dc94da62a0d1f9fb42937d555ac932566f822) dealt with your reported issue and will be available in the next release v6.1
